packages/frontend/src/hooks/useSqlQueryVisualization.ts

```diff
@@ -27,39 +30,73 @@ const useSqlQueryVisualization = ({
     initialState,
     sqlQueryMutation: { data },
 }: Args) => {
-    const sqlQueryDimensions: Record<FieldId, CompiledDimension> = useMemo(
+    const fields: {
+        sqlQueryDimensions: Record<FieldId, CompiledDimension>;
+        sqlQueryMetrics: Record<FieldId, CompiledMetric>;
+    } = useMemo(
         () =>
             Object.entries(data?.fields || []).reduce(
                 (acc, [key, { type }]) => {
-                    const dimension: CompiledDimension = {
-                        fieldType: FieldType.DIMENSION,
-                        type,
-                        name: key,
-                        label: friendlyName(key),
-                        table: SQL_RESULTS_TABLE_NAME,
-                        tableLabel: '',
-                        sql: '',
-                        compiledSql: '',
-                        hidden: false,
-                    };
-                    return { ...acc, [fieldId(dimension)]: dimension };
+                    if (type === DimensionType.NUMBER) {
```

Comment:
If column is numeric, consider it a metric instead of a dimension

Reply:
it is unusual to see Ids as metrics, but I guess it is OK 

![image](https://user-images.githubusercontent.com/1983672/179219782-be33e114-f0c6-4a02-856d-f469e8a07708.png)